vsts-gn.yml

```diff
@@ -32,7 +31,7 @@ phases:
   - bash: |
       cd src
       export CHROMIUM_BUILDTOOLS_PATH=`pwd`/buildtools
-      export SCCACHE_WRAPPER="`pwd`/libchromiumcontent/tools/sccache/aad2120/mac/sccache"
+      export SCCACHE_WRAPPER="`pwd`/electron/external_binaries/sccache"
       export SCCACHE_HELPER="`pwd`/libchromiumcontent/script/sccache"
```

Comment:
This line and the following one need to change, but I can address that in a separate PR.